### DISCUSSION TITLE
add next Plot Directory

### DISCUSSION THREAD

**Discussion by robertbigk** [guydavis/machinaris#191] [Q&A] [answered]
When checking plowman dir from machineries console the console

192 plots and i xch1 and 3 plots inside
![msedge_UBgQJa3y2l](https://user-images.githubusercontent.com/34956477/125980222-dfdae980-06ad-4402-a97a-103dffb38f1c.png)

But in machinaris shows only  192 plots
![msedge_s9vbDnzUMp](https://user-images.githubusercontent.com/34956477/125980300-e3fbee8c-3a53-42e1-ac0f-aeaad5269876.png)
![Uploading msedge_Wd0FTJnEBv.png…]()

machinaris not "see" additional plot folder

**Comment by guydavis** [comment, marked as answer]
Good day!  Please see this [wiki entry](https://github.com/guydavis/machinaris/wiki/FAQ#can-i-use-two-or-more-final-destination-directories-to-hold-my-plots) on how to configure more than on final plots directories within Machinaris.

If this answers your question, please mark the Discussion as Answered.  Otherwise, please provide additional details, showing exactly your `plots_dir` variable.  Unraid -> Advanced View toggle on, 'Show Additional Settings...' as well.

**Reply by robertbigk** [reply]
Please help me if the wiki entry is so simple I'm on spending a couple of hours asking ho to make work. I'm be not ask again and again if that works for me I can't find what I doing wrong.

first is not plotting or temp folder for 
Farming I got on this HDD already plots.

![msedge_rFtMIGknMf](https://user-images.githubusercontent.com/34956477/125990493-e069bed4-e046-40be-b975-f763a142b980.png)


My folders 
![msedge_f92IKPcfU6](https://user-images.githubusercontent.com/34956477/125990581-b264a322-528a-46be-9cee-0e6d384d7ba7.png)

![msedge_JnhlTEPF2G](https://user-images.githubusercontent.com/34956477/125990605-b725a90b-144d-424a-8b56-7524e98e82a7.png)

![msedge_JnhlTEPF2G](https://user-images.githubusercontent.com/34956477/125990654-0360b304-d767-4f78-9da1-a2c14a4caa25.png)
![msedge_KxUF2d5BP0](https://user-images.githubusercontent.com/34956477/125990703-351651ba-4a2a-4d68-80bc-c16887523818.png)


And in farming still have 192 plots no found 4 plots from xch1 drive 

![msedge_uQNnxKAcMB](https://user-images.githubusercontent.com/34956477/125990763-4cb665c9-5f9d-4c0e-ba87-d854fd6dbb84.png)

can You explain to me step by step no shortcuts for experience Linux user? I try many times and all-time fail and I even don't know why?

**Reply by guydavis** [reply]
Sorry that you are not having a good experience with Machinaris.  Let's review the setup.  Please ensure you have 'Show more settings' expanded so you see every Variable in the Machinaris config.  Look for any duplicates.

![image](https://user-images.githubusercontent.com/12936/125993413-24777991-091c-4513-8c56-2bb901392e4d.png)

Please provide the Edit view of the 'plots_dir' variable as shown here:

![image](https://user-images.githubusercontent.com/12936/125993573-542517f1-b6ed-450a-85cc-6c4a141fb6df.png)

Also, don't hesitate to join the [Discord server](https://discord.gg/mX4AtMTt87) for more interactive support.
